optimize/backend/src/main/java/io/camunda/optimize/service/security/AuthCookieService.java

```diff
@@ -148,17 +141,14 @@ public NewCookie createCookie(
     if (getAuthConfiguration().getCookieConfiguration().isSameSiteFlagEnabled()) {
       cookiePath = addSameSiteCookieFlag(cookiePath);
     }
-    return new NewCookie.Builder(cookieName)
-        .value(cookieValue)
-        .path(cookiePath)
-        .domain(null)
-        .version(1)
-        .comment(null)
-        .maxAge(-1)
-        .expiry(expiresAt)
-        .secure(isSecureScheme(requestScheme))
-        .httpOnly(true)
-        .build();
+    final int maxAge = (int) (expiresAt.toInstant().toEpochMilli() - System.currentTimeMillis());
```

Comment:
❓ The `maxAge` was previously hard coded to `-1` for this method. Why change to a calculated value?

Reply:
Good question - I could barely recall why ) The point is that previously there was this strange setup:
`        .maxAge(-1)
        .expiry(expiresAt)`
where we considered expiresAt, but only as a cookie expiration, while we were setting maxAge to -1.
Tomcat cookies do not support expiration setting (it is deprecated), but only maxAge. So I've got to translate the expiration into maxAge.